packages/deployments/contracts/contracts/shared/libraries/TypedMemView.sol

```diff
@@ -392,16 +392,6 @@ library TypedMemView {
     }
   }
 
-  /**
-   * @notice          Optimized type comparison. Checks that the 5-byte type flag is equal.
-   * @param left      The first view
-   * @param right     The second view
-   * @return          bool - True if the 5-byte type flag is equal
-   */
-  function sameType(bytes29 left, bytes29 right) internal pure returns (bool) {
-    return (left ^ right) >> TWENTY_SEVEN_BYTES == 0;
-  }
-
```

Comment:
We will not use this function in the future?

Reply:
if we do use it in the future, we can always return it from the source.

for now, we are not using it so may as well drop the function